.github/workflows/test.yaml

```diff
@@ -66,6 +66,7 @@ jobs:
           path: coverage/environments/${{ matrix.env.name }}
 
   test:
+    if: github.base_ref == 'master'
```

Comment:
Why would we not want to run tests when based against a release?

Reply:
We are running tests; this step is just collecting coverage. I started implementing CI coverage handling for release branches as well but it started touching upon multiple files and I felt it diluted the original idea of the PR too much. I'd rather implement it in another PR - if it needs to be implemented at all.

Reply:
Ah, of course!

Reply:
Really want to change the name of that stage.